WEBROOOOT/portfolio.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	
+	<link rel="stylesheet" href="style.css">
+
+	 <title>P0RTFOLIO</title>
+	</head>
+<nav> 
+
+		<a href="index.html">(It's All) ALL ABOUT ME</a> 
+	 	<a href="portfolio.html">PORTF0LIO</a> 
+
+
+	 </nav>
+
+</center>
+
+</head>
+
+<body>
+
+<h1> <img src="77eyes.gif"></h1>
+<br> 
+<br>
+<br>
+<h2><img src="LILSPARKLYTREE.gif"></h2>
+
+ <h3> ///SOME RECENT PR0JECTS/// </h3>
+
+ 	<style> 
+ 	p {color: black;}
+ <p> 
+Works 2010-2014-ish leaning largely towards the retinal 
+
+</p>
+
+
+	</style>
+
+		<img src="projects.png">
```

Comment:
Something to think about if you revisit working on this page - how would you add multiple projects to this page, with each having their own image?